### ISSUE TITLE
Bug: toybox warning/broken

### ISSUE THREAD

**Issue by gingerbeardman** [macvogelsang/pd-options#1]
`toybox` integration is currently broken. i don't know enough about it to fix it. 

Maybe @didiermalenfant can help?

```
❯ toybox add options
Added a dependency for 'options@1'.
❯ toybox update
Resolving dependencies...
Installed 'github.com/macvogelsang/pd-options@(>=1.0.0 <2.0.0)' -> 1.1.1.
Warning: Could not find file 'options.lua' to import in 'pd-options'.
Finished.
```

might be a confusion between `options` and `pd-options`? maybe?

ps: yes, Mac, I'm using this!

**Comment by DidierMalenfant** [comment]
I'll take a look in the morning!

**Comment by DidierMalenfant** [comment]
This is fixed in #4 

Until the fix is merged, you can create a local repo with the fix on your machine, let's say in `~/Desktop/pd-options` and then do `toybox -l ~/Desktop update`. This will use any toyboxes it find locally in that folder instead of using the official GitHub ones, letting you use the fix for now.

When the fix is merged, just do a regular `toybox update` and this this revert everything to using online toyboxes.

I use `-l` (local) a lot when developing toyboxes. You can just work on them locally and they are plugged in the main project with a softlink on macOS and Linux (you don't even have to keep on updating when making changes). It's awesome.

**Comment by gingerbeardman** [comment]
Thanks! Neat trick.
